tdp/cli/commands/queries.py

```diff
@@ -39,3 +40,20 @@ def get_latest_success_service_version_query():
         )
         .order_by(desc(ServiceLog.deployment_id))
     )
+
+
+def get_deployment(session_class, deployment_id):
+    query = (
+        select(DeploymentLog)
+        .options(
+            joinedload(DeploymentLog.services), joinedload(DeploymentLog.operations)
+        )
+        .where(DeploymentLog.id == deployment_id)
+        .order_by(DeploymentLog.id)
+    )
+
+    with session_class() as session:
+        deployment_log = session.execute(query).unique().scalar_one_or_none()
+        if deployment_log is None:
+            raise click.ClickException(f"Deployment id {deployment_id} does not exist")
```

Comment:
We should raise the right exception instead of a ClickException, ClickException should only be raised from command files.